### DISCUSSION TITLE
Docker build failure on Ubuntu 18.04 host

### DISCUSSION THREAD

**Discussion by gitfool** [community/community#25884] [Actions]
<p>I’m hitting a strange build issue which seems to be down to the differences between Ubuntu hosts:</p>
<ul>
<li>
<a href="https://github.com/gitfool/cake-docker/runs/240881669#step:6:270" rel="nofollow noopener">https://github.com/gitfool/cake-docker/runs/240881669#step:6:270</a> (fails)</li>
<li>
<a href="https://github.com/gitfool/cake-docker/runs/240940390#step:6:268" rel="nofollow noopener">https://github.com/gitfool/cake-docker/runs/240940390#step:6:268</a> (succeeds)</li>
</ul>
<p>The only difference is the failing build is running on Ubuntu 18.04, whereas the succeeding build is running on Ubuntu 16.04. This is very strange because that part is failing inside of docker build, during the <code>RUN</code> command, which should have a consistent “environment” everywhere. I presume this is due to Docker Engine differences, unless I’m missing something?</p>

**Comment by ethomson** [comment, marked as answer]
<p>Hi <a class="mention" href="/gitfool">@gitfool</a>, apologies for the inconvenience.  It looks like there was an <a href="https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1843018" rel="nofollow noopener">issue with a kernel update to Ubuntu 18.04 that caused this problem</a>.  We’ll investigate to see if we can update the kernel version to resolve this issue.</p>
